src/kOS/Screen/TermWindow.cs

```diff
@@ -47,13 +52,14 @@ public class TermWindow : KOSManagedWindow
         public TermWindow()
         {
             IsPowered = true;
-            windowRect = new Rect(60, 50, 470, 405);
+            windowRect = new Rect(50, 60, 0, 0); // will get resized later in AttachTo().
         }
 
         public void Awake()
         {
             LoadTexture("GameData/kOS/GFX/font_sml.png", ref fontImage);
             LoadTexture("GameData/kOS/GFX/monitor_minimal.png", ref terminalImage);
+            LoadTexture("GameData/kOS/GFX/resize-button.png", ref resizeButtonImage);
```

Comment:
For some reason I cannot fathom, this is getting rendered on the button as just a tiny little red squiggle.  That's not what the image file actually looks like.